Build error on nan_point.h

This error should be a regression after merging #114 

https://travis-ci.org/01org/widl-nan/jobs/226802447

```
make: Entering directory `/home/travis/build/01org/widl-nan/test/static-member/build'
  CXX(target) Release/obj.target/widlNanAddon/addon.o
../addon.cpp:3:28: fatal error: gen/nan__point.h: No such file or directory
 #include "gen/nan__point.h"
```

--- a/test/array/array.widl
+++ b/test/array/array.widl
@@ -2,17 +2,13 @@
 // Use of this source code is governed by a MIT-style license that can be
 // found in the LICENSE file.
 
-[
-Constructor
-]
+[Constructor]
 interface Point {
     attribute long x;
     attribute long y;
 };
 
-[
-Constructor
-]
+[Constructor]
 interface Canvas {
   attribute double[] coordinates;
   attribute DOMString[] codecs;

--- a/test/attributes/attributes.widl
+++ b/test/attributes/attributes.widl
@@ -8,8 +8,7 @@
 [
 Constructor,
 Constructor(DOMString name),
-Constructor(DOMString name, unsigned short age)
-]
+Constructor(DOMString name, unsigned short age)]
 interface Animal {
 
   // A simple attribute that can be set to any string value.

--- a/test/callback/callback.widl
+++ b/test/callback/callback.widl
@@ -8,9 +8,7 @@
 callback AsyncOperationCallback = void (DOMString status);
 callback AnotherOperationCallback = void (DOMString type, double value);
 
-[
-Constructor
-]
+[Constructor]
 interface AsyncOperations {
   void performOperation(AsyncOperationCallback whenFinished);
   void performAnotherOperation(AnotherOperationCallback whenStarted);

--- a/test/constructor/constructor.widl
+++ b/test/constructor/constructor.widl
@@ -5,8 +5,7 @@
 [
 Constructor,
 Constructor(long radius),
-Constructor(DOMString url, long radius)
-]
+Constructor(DOMString url, long radius)]
 interface MyClass {
   attribute long radius;
   attribute DOMString url;
@@ -20,8 +19,7 @@ interface NoConstructorClass {
 [
 Constructor,
 Constructor(MyClass obj),
-Constructor(double diameter, MyClass obj)
-]
+Constructor(double diameter, MyClass obj)]
 interface ContainerClass {
   readonly attribute MyClass embedded;
   readonly attribute double diameter;

--- a/test/dictionary-enum/dictionary-enum.widl
+++ b/test/dictionary-enum/dictionary-enum.widl
@@ -16,9 +16,7 @@ dictionary NodeAttr {
   ComputingDeviceType type;
 };
 
-[
-Constructor
-]
+[Constructor]
 interface ComputingDeviceNetwork {
   readonly attribute DOMString name;
   readonly attribute double quantity;

--- a/test/dictionary/dictionary.widl
+++ b/test/dictionary/dictionary.widl
@@ -3,8 +3,7 @@
 // found in the LICENSE file.
 [
 Constructor,
-Constructor(long x, long y)
-]
+Constructor(long x, long y)]
 interface Point {
   attribute long x;
   attribute long y;
@@ -28,9 +27,7 @@ dictionary PolygonDrawOption {
   DOMString? color = "black";
 };
 
-[
-Constructor
-]
+[Constructor]
 interface Painter {
   void drawText(DOMString text, PaintOptions options);
   bool checkOptions(PaintOptions options);

--- a/test/enum/enum.widl
+++ b/test/enum/enum.widl
@@ -7,9 +7,7 @@
 
 enum MealType { "rice", "noodles", "other" };
 
-[
-Constructor
-]
+[Constructor]
 interface Meal {
   attribute MealType type;
   attribute float size;     // in grams

--- a/test/event-emitter/promise.widl
+++ b/test/event-emitter/promise.widl
@@ -3,9 +3,7 @@
 // found in the LICENSE file.
 //
 
-[
-Constructor
-]
+[Constructor]
 interface Meal {
   readonly attribute boolean isRawMeal;
 

--- a/test/method_overload/method_overload.widl
+++ b/test/method_overload/method_overload.widl
@@ -13,9 +13,7 @@
 // };
 
 // Node params were removed because there is no `Node` type in Node.js :)
-[
-Constructor
-]
+[Constructor]
 interface B {
   /* f1 */ DOMString f(DOMString w);
   /* f2 */ DOMString f(long w, float x);

--- a/test/optional-argument/optional-argument.widl
+++ b/test/optional-argument/optional-argument.widl
@@ -5,9 +5,7 @@
 // Extracted from https://www.w3.org/TR/WebIDL/#dfn-optional-argument-default-value
 // W3C Candidate Recommendation 19 April 2012
 
-[
-Constructor
-]
+[Constructor]
 interface ColorCreator {
   DOMString createColor(float r, float g, float b, optional float alpha);
   DOMString createColor2(float r, float g, float b, optional float alpha = 0.75);

--- a/test/primitives/primitives-attributes.widl
+++ b/test/primitives/primitives-attributes.widl
@@ -2,9 +2,7 @@
 // Use of this source code is governed by a MIT-style license that can be
 // found in the LICENSE file.
 //
-[
-Constructor
-]
+[Constructor]
 interface PrimitivesAttributes {
   attribute byte vByte;
   attribute octet vOctet;

--- a/test/primitives/primitives-param.widl
+++ b/test/primitives/primitives-param.widl
@@ -2,9 +2,7 @@
 // Use of this source code is governed by a MIT-style license that can be
 // found in the LICENSE file.
 
-[
-Constructor
-]
+[Constructor]
 interface PrimitivesParam {
    attribute boolean vBoolean;
    attribute float vFloat;

--- a/test/primitives/primitives-return.widl
+++ b/test/primitives/primitives-return.widl
@@ -2,9 +2,7 @@
 // Use of this source code is governed by a MIT-style license that can be
 // found in the LICENSE file.
 
-[
-Constructor
-]
+[Constructor]
 interface PrimitivesReturn {
     attribute boolean vBoolean;
     attribute float vFloat;

--- a/test/promise/promise.widl
+++ b/test/promise/promise.widl
@@ -3,9 +3,7 @@
 // found in the LICENSE file.
 //
 
-[
-Constructor
-]
+[Constructor]
 interface Meal {
   readonly attribute boolean isRawMeal;
 

--- a/test/static-member/static-member.widl
+++ b/test/static-member/static-member.widl
@@ -4,8 +4,7 @@
 
 [
 Constructor,
-Constructor(long x, long y)
-]
+Constructor(long x, long y)]
 interface Point {
   attribute long x;
   attribute long y;
@@ -14,8 +13,7 @@ interface Point {
 [
 Constructor,
 Constructor(Point pt1, Point pt2),
-Constructor(long x1, long y1, long x2, long y2)
-]
+Constructor(long x1, long y1, long x2, long y2)]
 interface Line {
   attribute Point startPt;
   attribute Point endPt;

--- a/test/timer/timer.widl
+++ b/test/timer/timer.widl
@@ -1,6 +1,4 @@
-[
-Constructor
-]
+[Constructor]
 interface TimerTester {
   attribute long taskCounter;
 

--- a/test/typedarray/typedarray.widl
+++ b/test/typedarray/typedarray.widl
@@ -1,6 +1,4 @@
-[
-Constructor
-]
+[Constructor]
 interface FakeImage {
   readonly attribute Uint8Array rawBuffer;
   readonly attribute unsigned long length;

